README.md

```diff
@@ -21,37 +21,19 @@
 
 ##### On Mac
 
-```bash
-# install go, set $GOPATH
-brew install go
-
-# install glide for go dependencies
-brew install glide
+Install Yarn for JavaScript dependencies. This will also install Node.js if necessary.
 
-# install yarn for js dependencies
+```bash
 brew install yarn
 ```
 
 ##### On Windows
 
-Install [Go](https://golang.org/dl/)
-
-Install [Glide](https://github.com/Masterminds/glide/releases)
-
 Install [Yarn](https://yarnpkg.com/lang/en/docs/install/#windows)
 
-#### Install Gaia
+#### Install Cosmos SDK
```

Comment:
is it more accurate to say `gaiacli` and `gaiad`?

Reply:
I updated it. Does that sound better?